vignettes/GMAT-GMST.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Example: Calculating GMST from GMAT in Hector"
+author: "Leeya Pressburger"
+date: "2021-08-18"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Example: Carbon tracking with Hector}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Introduction
+
+The global mean surface temperature (GMST) is the average of the air temperature over land and the temperature at the surface of the ocean. This is a very useful metric for observing global warming or global cooling. However, Hector does not produce GMST as an output at this time. Hector does measure the global mean air temperature (GMAT), which we can use to back-calculate GMST. 
```

Comment:
Maybe I'm just being a dorky scientist, but a citation at the end of that first sentence would be good.

"...for observing global temperature changes."  ?